packages/mobile/src/identity/verification.test.ts

```diff
@@ -106,11 +106,21 @@ const mockActionableAttestations: ActionableAttestation[] = [
 ]
 
```

Comment:
I think it also makes sense to add some tests that mock the `getAttestationStat` call and test that `getAttestationStatus` behaves correctly.

Reply:
ya -- I was looking for examples of contractKit wrapper tests and thought it was weird I didn't see a test directory.  just realized all the tests are alongside the files in the wrappers directory 🤦‍♀️

i'll add some testing there